compiler/testData/diagnostics/tests/platformTypes/nullabilityWarnings/notNullAssertion.fir.kt

```diff
@@ -23,18 +23,18 @@ fun test() {
     val platformJ = J.staticJ
 
     if (platformNN != null) {
-        platformNN!!
+        platformNN<!UNNECESSARY_NOT_NULL_ASSERTION!>!!<!>
     }
 
     if (platformN != null) {
-        platformN!!
+        platformN<!UNNECESSARY_NOT_NULL_ASSERTION!>!!<!>
     }
 
     if (platformJ != null) {
-        platformJ!!
+        platformJ<!UNNECESSARY_NOT_NULL_ASSERTION!>!!<!>
     }
 
-    platformNN!!
+    platformNN<!UNNECESSARY_NOT_NULL_ASSERTION!>!!<!>
     platformN!!
-    platformJ!!
+    platformJ<!UNNECESSARY_NOT_NULL_ASSERTION!>!!<!>
```

Comment:
This one is wrong, but it's due to the wrong DFA, which still has smartcast info on this variable. Maybe related to https://youtrack.jetbrains.com/issue/KT-44308

Reply:
No longer reported on top of https://github.com/JetBrains/kotlin/pull/4269, and `// FIR_IDENTICAL` now.